### ISSUE TITLE
Accessibility: Year cells should have scope=row

### ISSUE THREAD

**Issue by dougmet** [open-sdg/open-sdg#106]
Moved from ONSdigital/sdg-indicators#1893

For screen readers a table is much easier to navigate if columns and rows are labelled. We're lucky because all of our datasets are time series so the Year column can be used as a row label (think about how quarters would affect this). To do this we change them to `<th>` tags.

```
<th scope="row">2015</th>
```

as here https://webaim.org/techniques/tables/data#th
